velox/exec/tests/MarkDistinctTest.cpp

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/exec/tests/utils/AssertQueryBuilder.h"
+#include "velox/exec/tests/utils/OperatorTestBase.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::test;
+using namespace facebook::velox::exec::test;
+
+class MarkDistinctTest : public OperatorTestBase {
+ public:
+  void markDistinctTest(const VectorPtr& base) {
+    std::vector<RowVectorPtr> vectors;
+
+    const vector_size_t baseSize = base->size();
```

Comment:
This seems like a very simple distribution for testing distinctness. Can we do something that involves any kind of random values generation ?

Reply:
I'm not sure I agree that this is a simple distribution. I think random values would be outside the scope of this PR. We rely on GroupingSet to compute uniqueness correctly.